addons/vagrant/playbooks/linux_servers/samba4ad.yml

```diff
@@ -0,0 +1,201 @@
+- hosts: service_samba4ad
+  name: Manage Samba service
+  become: True
+
+  collections:
+    - debops.debops
+    - debops.roles01
+    - debops.roles02
+    - debops.roles03
+
+  #roles:
+  #  - role: samba
+
+  tasks:
+  - name: Disable IPV6
+    ansible.posix.sysctl:
+      name: "{{ item }}"
+      value: '1'
+      sysctl_set: true
+    loop:
+    - "net.ipv6.conf.all.disable_ipv6"
+    - "net.ipv6.conf.default.disable_ipv6"
+    - "net.ipv6.conf.lo.disable_ipv6"
+
+  - name: Install Samba packages
```

Comment:
The title is not correct, I think these packages are prerequisite to install Samba packages.

Reply:
done